### PR TITLE
Update 2022-06-05-ccss-summer-dev-projects-introduction.md

### PR DESCRIPTION
**Pull request by xxArchitect**

### CODE REVIEW

File: content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md [hunk 1]
```diff
@@ -31,3 +30,14 @@ part of the Rust Gamedev Working Group, and the team lead of the Rust CTCFT
 team. I’m also one of the core developers on Veloren, an open source game
 similar to Minecraft. In the Rust communty, I run a monthly game dev meetup, and
 co-edit the monthly game dev newsletter.
+
+### Vasily Inkovskiy (@xxArchitect)
+
+Hi! I’m Vasily 👋
+
+I’m a computer science student at Carleton University in  Ottawa, Canada 🇨🇦.
```
Comment (AngelOnFira) [inline content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md]:
```suggestion
I’m a computer science student at Carleton University in Ottawa, Canada 🇨🇦. 
```

File: content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md [hunk 2]
```diff
@@ -31,3 +30,14 @@ part of the Rust Gamedev Working Group, and the team lead of the Rust CTCFT
...
 similar to Minecraft. In the Rust communty, I run a monthly game dev meetup, and
 co-edit the monthly game dev newsletter.
+
+### Vasily Inkovskiy (@xxArchitect)
+
+Hi! I’m Vasily 👋
+
+I’m a computer science student at Carleton University in  Ottawa, Canada 🇨🇦. 
+My concentration is Software Engineering. I'm intrested in many topics, including 
+web development, UI/UX design, operating systems, and mutlithreading.
```
Comment (AngelOnFira) [inline content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md:114]: This is super neat! I love this stuff too 🎉

Comment (igortascu) [inline content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md:114]: very nice

File: content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md [hunk 3]
```diff
@@ -12,7 +12,6 @@ author: Forest Anderson
 categories:
 - blog
 ---
-
```
Comment (AngelOnFira) [inline content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md:15]: Please revert this deletion

**Pull request closed without merging**